### ISSUE TITLE
Option to keep dns rewrites per client but not global settings

### ISSUE THREAD

**Issue by t3dium** [AdguardTeam/AdGuardHome#4425]
On some clients, until i'm able to have per client blocklists I have to disable global settings (and therefore filters).

This also means that I lose dns-rewrites however, so it would be nice to have that as a separate option 
![image](https://user-images.githubusercontent.com/83690012/159940783-d168d680-d556-4136-85c2-3ee2dc21031c.png)

**Comment by EugeneOne1** [comment]
@t3dium, hello and thanks for a suggestion.  Perhaps, we may add the "enable protection" checkbox here.  Anyway, we'll consider adding this.

**Comment by Kylindrias** [comment]
I'd also like to see this implemented.

I've had to disable Global Blocklists as it was causing too many issues with the Mrs. So rather than opt out of AdGuard I want to make it opt in and only have certain IP addresses use the filters. This means the entire network can benefit from DNS over TLS, DNS over HTTPS, etc, but Ads will only be blocked on specific devices.

Unfortunately, the current options available breaks DNS-rewrites which I want to keep as they're all for my own private network.

Is it possible for DNS rewrites to be a separate option entirely as a global setting under General Settings?

The option is "Block domains using filters and hosts files", but I'm not blocking domains, in fact, unchecking the option is what is blocking the domains so the option is having the opposite effect.

**Comment by Michaelcombs** [comment]
Yes, please add the 'Disable Protection' feature to the client specific settings which preserves DNS rewrites.

**Comment by Lampone** [comment]
Is there a Option, or a other workaround by now?

**Comment by Kylindrias** [comment]
> Is there a Option, or a other workaround by now?

The work around is to just use your router for your internal DNS entries.

In your AdGuard settings you can put the following as a new line in Settings > DNS Settings > Upstream DNS Servers

`[/your.domain/]192.168.1.1`

Then any time you want to access your.domain instead of asking Google, Cloudflare, Quad9, etc, it will ask your router.
